### PR TITLE
Signup back button, errors

### PR DESCRIPTION
**Pull request by emilyzhong**
## Signup back button, errors
For artist + buyer sign up:
* Include back button on second page of form (terms and conditions)
* Display form validation error (incorrect email domain, password length, password confirmation)

### Related PRs
https://github.com/calblueprint/SFAI/pull/65

### Migrations
None

### Tests Performed, Edge Cases
None

### Screenshots
_the ending loading cut off but after signup it should redirect to the landing page_
error

CC: @by-co

### CODE REVIEW

File: app/assets/javascripts/signin.js [hunk 1]
```diff
@@ -18,57 +18,64 @@ enableSubmit = (el) => {
...
+		return true;
 	}
+}
 
-	/* Validate password confirmation is minimum length */
-	if (!passwordConfirmation.val() ||
-		passwordConfirmation.val().length < 6) {
-		valid = false;
+validateConfirmation = (password, confirmation, showError) => {
+	console.log('hnng');
```
Comment (by-co) [inline app/assets/javascripts/signin.js]: stray console?
